flutter/shell/platform/tizen/tizen_autofill.h

```diff
@@ -0,0 +1,74 @@
+// Copyright 2023 Samsung Electronics Co., Ltd. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+#ifndef EMBEDDER_TIZEN_AUTOFILL_H_
+#define EMBEDDER_TIZEN_AUTOFILL_H_
+
+#include <functional>
+#include <memory>
+#include <optional>
+#include <string>
+#include <vector>
+
+#include <autofill.h>
+
+struct AutofillItem {
+  autofill_hint_e hint_;
+  bool sensitive_data_;
+  std::string label_;
+  std::string id_;
+  std::string value_;
+};
+
+class TizenAutofill {
+ public:
+  static TizenAutofill& GetInstance() {
+    static TizenAutofill instance = TizenAutofill();
+    return instance;
+  }
+
+  void RequestAutofill(std::vector<std::string> hints, std::string id);
+
+  void RegisterAutofillItem(std::string view_id, AutofillItem item);
+
+  void StoreResponseItem(std::unique_ptr<AutofillItem> item) {
+    response_items_.push_back(move(item));
+  }
+
+  void SetPopupCallback(std::function<void()> callback) {
+    popup_callback_ = callback;
+  }
```

Comment:
I think that using prefix 'On'  and remove postfix 'Callback' seems more consistent.
```c++
  void SetOnPopup(std::function<void()> on_popup) {
    on_popup_ = on_popup;
  }
```
...
```c++
  void SetOnCommit(std::function<void(std::string)> on_commit) {
      on_commit_ = on_commit;
  }
 
  void OnCommit(std::string str) { on_commit_(str); }
```

Reply:
Thank you for the good suggestion. it's modified.